src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -65,6 +66,7 @@ public enum UnreferencedTypesHandlingOption
             bool skipSerializationFormatXml,
             bool disablePaginationTopRenaming,
             bool generateModelFactory,
+            bool publicDiscriminatorProperty,
```

Comment:
Not quite understand what true & false mean for this. Do you mean  EnablePublicDiscriminatorProperty?  Maybe you should find a better name.

Reply:
We use this configuration like this:
```
public-discriminator-property: true
```

which means we will have a public discriminator property, otherwise we do not. Which part do we have the confusion?

Reply:
And in the generator code, we use this like this:
```
var accessibilityOfDiscriminatorProperty = Configuration.PublicDiscriminatorProperty ? Public : Internal;
```
I think this is clear enough